snippets/javascript.json

```
@@ -653,5 +653,868 @@
     "prefix": [
       "callout3DLargeProps"
     ]
+  },
+  "Add CSV Layer": {
+    "body": [
+    "CSVLayer({",
+    "\turl: \"${1:https://earthquake.usgs.gov/earthquakes/feed/v1.0/summary/2.5_week.csv}\",",
+    "\tcopyright: \"${2:USGS Earthquakes}\",",
+    "\tpopupTemplate: ${3:template}",
+    "});",
+    "map.add(csvLayer);"
+    ],
+    "description": "Add a CSVLayer with a URL",
+    "prefix": "csvLayer"
```

```suggestion
    "prefix": "CSVLayer"
```
Prefix on **Class constructors** should match the SDK capitalization according to [the conventions](https://github.com/Esri/arcgis-js-vscode-snippets/blob/master/CONTRIBUTING.md#prefix)